src/selectors/visibleBreakpoints.js

```diff
@@ -0,0 +1,30 @@
+import { getSelectedSource } from "../reducers/sources";
+import { isGeneratedId } from "devtools-source-map";
+
+function getLocation(breakpoint, isGeneratedSource) {
+  return isGeneratedSource
+    ? breakpoint.generatedLocation || breakpoint.location
+    : breakpoint.location;
+}
+
+export default function getVisibleBreakpoints(state: OuterState) {
```

Comment:
This will get the breakpoints for a file at the "correct" location regardless of original / generated